includes/class-newspack-popups-model.php

```diff
@@ -211,12 +211,13 @@ public static function set_popup_options( $id, $options ) {
 	/**
 	 * Retrieve all inline popups.
 	 *
+	 * @param  boolean $include_unpublished Whether to include unpublished posts.
 	 * @return array Inline popup objects.
 	 */
-	public static function retrieve_inline_popups() {
+	public static function retrieve_inline_popups( $include_unpublished = false ) {
```

Comment:
This method is no longer used with the argument

Reply:
Good catch, removed in a603104.